src/System.Text.Json/src/System/Text/Json/JsonUtf8Reader.cs

```diff
@@ -0,0 +1,1274 @@
+                    }
+                }
+            }
+
+            if (marker == JsonConstants.ListSeperator)
+            {
+                _consumed++;
+                _lineBytePosition++;
+
+                if (_consumed >= (uint)_buffer.Length)
+                {
+                    if (IsLastSpan)
+                    {
+                        _consumed--;
+                        _lineBytePosition--;
+                        ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.ExpectedStartOfPropertyOrValueNotFound);
+                    }
+                    else
+                        return ConsumeTokenResult.IncompleteRollback;
+                }
+                byte first = _buffer[_consumed];
+
+                if (first <= JsonConstants.Space)
+                {
+                    SkipWhiteSpace();
+                    // The next character must be a start of a property name or value.
+                    if (!HasMoreData(ExceptionResource.ExpectedStartOfPropertyOrValueNotFound))
+                        return ConsumeTokenResult.IncompleteRollback;
+                    first = _buffer[_consumed];
+                }
+
+                if (_inObject)
+                {
+                    if (first != JsonConstants.Quote)
+                        ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.ExpectedStartOfPropertyNotFound, first);
+                    return ConsumePropertyName() ? ConsumeTokenResult.Success : ConsumeTokenResult.IncompleteRollback;
+                }
+                else
+                    return ConsumeValue(first) ? ConsumeTokenResult.Success : ConsumeTokenResult.IncompleteRollback;
+            }
+            else if (marker == JsonConstants.CloseBrace)
+                EndObject();
+            else if (marker == JsonConstants.CloseBracket)
+                EndArray();
+            else
+                ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.FoundInvalidCharacter, marker);
+            return ConsumeTokenResult.Success;
+        }
+
+        private bool SkipComment()
+        {
+            //Create local copy to avoid bounds checks.
+            ReadOnlySpan<byte> localCopy = _buffer.Slice(_consumed + 1);
+
+            if (localCopy.Length > 0)
+            {
+                byte marker = localCopy[0];
+                if (marker == JsonConstants.Solidus)
+                    return SkipSingleLineComment(localCopy.Slice(1), out _);
+                else if (marker == '*')
+                    return SkipMultiLineComment(localCopy.Slice(1), out _);
+                else
+                    ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.ExpectedStartOfValueNotFound, marker);
+            }
+
+            if (IsLastSpan)
+                ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.ExpectedStartOfValueNotFound, JsonConstants.Solidus);
```

Comment:
Is Solidus the term the JSON spec uses for a slash?  I know it's historically valid, but it's not common to see, making it more confusing (at least to me) than just "slash".

Reply:
Slash/Backslash instead of Solidus/Reverse Solidus then? I am fine with that.

Reply:
> Is Solidus the term the JSON spec uses for a slash?

Yes, it uses that term: https://tools.ietf.org/html/rfc8259#section-7

Does that change your recommendation?

Reply:
Ok. I expect it just means that a majority of people that read the code will need to look it up.